### PR TITLE
A tiny js world

### PR DESCRIPTION
**Pull request by Shrai-dev**
### A tiny js world (pre-OOP)

[Demo](https://shrai-dev.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/Shrai-dev/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Shrai-dev/a-tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+   species,
+   name,
+   age,
+   legs,
+   hands,
+   friends,
+   saying
+}) => {
+   return 'I am a ' + `${species}` + '. My name is ' + `${name}` + '. I am ' + `${age}` + ' years old. I have ' + `${legs}` + ' legs and ' + `${hands}` + ' hands. My friends are: ' + `${friends}` + '. I say: ' + `${saying}`;
+}
```
Comment (OleksiyRudenko) [inline submissions/Shrai-dev/a-tiny-js-world/index.js:68]: The options below are equivalent and both are more cosistent than the current approach combining string concatenation and template strings/literals:
- ` 'I am a ' + species + '. My name is ' + name `
- `` `I am ${species}. My name is ${name}` ``

The latter is generally preferred. Check [the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to master it.

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@Shrai-dev good job!
Just stick to either of methods of a string composition for the sake of consistency.

File: submissions/Shrai-dev/a-tiny-js-world/index.js (hunk 1 above)
Comment (Shrai-dev) [reply submissions/Shrai-dev/a-tiny-js-world/index.js]: Done.

**Review by OleksiyRudenko — APPROVED** [review]
👍

**Pull request merged**